### PR TITLE
update zh-tw translation

### PR DESCRIPTION
**Pull request by vrabe**
## What does this PR do?

Update traditional Chinese (zh-tw) translation.

## Test Plan

No

## Related PRs and Issues

Previous translation update: #112, #688, #2134

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by gewenyu99** [comment]
您好，这些改动是有什么讲究的吗？我自己是小时候在大陆生活过一段时间的，但是对于台湾的常用国语翻译不是很了解。

有没有什么您引用的文献之类的，我可以对照参考来做Review，会简单一点。

谢谢!

**Comment by vrabe** [comment]
國名參照的是[CLDR](https://unicode-org.github.io/cldr-staging/charts/41/summary/zh_Hant.html#73c7c09de32184)。除了已經改名的北馬其頓(North Macedonia)外，基本上都是使用那邊的翻譯。

另外也有參考維基百科的[字詞轉換](https://zh.wikipedia.org/wiki/Module:CGroup/%E5%9C%B0%E5%90%8D)，我沒細看，但大致差不多。

File: app/config/locale/translations/zh-tw.json [hunk 1]
```diff
@@ -111,117 +117,117 @@
...
+    "countries.kg": "吉爾吉斯",
     "countries.kh": "柬埔寨",
-    "countries.ki": "基裏巴斯",
-    "countries.kn": "聖基茨和尼維斯",
+    "countries.ki": "吉里巴斯",
+    "countries.kn": "聖克里斯多福及尼維斯",
     "countries.kr": "韓國",
     "countries.kw": "科威特",
-    "countries.la": "老撾",
+    "countries.la": "寮國",
```
Comment (gewenyu99) [inline app/config/locale/translations/zh-tw.json:132]:
```suggestion
    "countries.la": "寮人",
```

Source: https://www.mofa.gov.tw/CountryInfo.aspx?CASN=0&n=487&sms=33&s=157

File: app/config/locale/translations/zh-tw.json [hunk 2]
```diff
@@ -111,117 +117,117 @@
...
     "countries.pa": "巴拿馬",
     "countries.pe": "秘魯",
     "countries.ph": "菲律賓",
-    "countries.pw": "帕勞",
-    "countries.pg": "巴布亞新幾內亞",
+    "countries.pw": "帛琉",
+    "countries.pg": "巴布亞紐幾內亞",
     "countries.pl": "波蘭",
-    "countries.kp": "北朝鮮",
+    "countries.kp": "北韓",
```
Comment (gewenyu99) [inline app/config/locale/translations/zh-tw.json:178]: 这边建议一起研究一下这个要不要改。这个比较敏感，而且与中华民国外交部使用的名称不符
https://www.mofa.gov.tw/CountryInfo.aspx?CASN=5&n=5&sms=33&s=48

File: app/config/locale/translations/zh-tw.json [hunk 3]
```diff
@@ -111,117 +117,117 @@
...
-    "countries.sr": "蘇裏南",
+    "countries.st": "聖多美普林西比",
+    "countries.sr": "蘇利南",
     "countries.sk": "斯洛伐克",
-    "countries.si": "斯洛文尼亞",
+    "countries.si": "斯洛維尼亞",
     "countries.se": "瑞典",
-    "countries.sz": "斯威士蘭",
-    "countries.sc": "塞舌爾",
+    "countries.sz": "史瓦濟蘭",
```
Comment (gewenyu99) [inline app/config/locale/translations/zh-tw.json:201]: Actually, `史瓦帝尼`
According to: https://www.mofa.gov.tw/CountryInfo.aspx?CASN=0&n=487&sms=33&s=90

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Reviewed translations against The Republic of China, Ministry of Foreign Affairs website. I want to make sure we avoid mistakes here and follow official naming guidelines.
https://www.mofa.gov.tw/CountryAreaInfo.aspx?casn=5&n=5&sms=33

File: app/config/locale/translations/zh-tw.json (hunk 1 above)
Comment (vrabe) [reply app/config/locale/translations/zh-tw.json:132]: The full name is 寮人民民主共和國. But we don't call it 寮人. You can check the tabs in that page.

File: app/config/locale/translations/zh-tw.json (hunk 3 above)
Comment (vrabe) [reply app/config/locale/translations/zh-tw.json:201]: Yes, the country has renamed to Eswatini. But the original text is [Swaziland](https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L201). So I keep using `史瓦濟蘭`. I can also use `史瓦帝尼` as you recommended.

File: app/config/locale/translations/zh-tw.json (hunk 2 above)
Comment (vrabe) [reply app/config/locale/translations/zh-tw.json:178]: 兩個名稱都有人用，以我的感覺北韓一詞比較常用，所以這樣改。當然也不能說原來的翻譯就是錯的，兩個名稱都可以理解，只是用北韓會比較熟悉。如果要改回去我也是可以接受的。

**Comment by vrabe** [comment]
@gewenyu99 Thanks for reviewing! You must have checked a lot pages to review my change.

File: app/config/locale/translations/zh-tw.json (hunk 3 above)
Comment (gewenyu99) [reply app/config/locale/translations/zh-tw.json]: I'd just update it at this point :D

I think both names would work, but for the sake of being with the times, let's use the new name,

File: app/config/locale/translations/zh-tw.json (hunk 2 above)
Comment (gewenyu99) [reply app/config/locale/translations/zh-tw.json:178]: 北韩就北韩吧，大概率不会有朝鲜用户用得到这个的 这个不改了

File: app/config/locale/translations/zh-tw.json (hunk 1 above)
Comment (gewenyu99) [reply app/config/locale/translations/zh-tw.json:132]: 彳亍。只要符合你们口语化用法就可以了。

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**